src/main/java/com/metamx/common/StreamUtils.java

```diff
@@ -33,34 +40,38 @@
 public class StreamUtils
 {
   // The default buffer size to use (from IOUtils)
-  private static final int DEFAULT_BUFFER_SIZE = 1024 * 4;
+  protected static final int DEFAULT_BUFFER_SIZE = 1024 * 4;
```

Comment:
It was used in FileUtils and CompressionUtils in older versions of this PR. Will revert.